content/issues/72.markdown

```diff
@@ -4,7 +4,7 @@ undefined
 
 ## Featured
 
-undefined
+[Haskell Study Startup](https://github.com/sjsyrek/haskell-study-startup) - Launch your own Haskell study group
```

Comment:
Because I'm a pedant, I have some feedback about this. In decreasing severity:

1. This section is an unordered list of links. Each element should follow this format:

    ``` markdown
    -   [Title in sentence case, less than 80 words](https://example.com/the-url)

        > An excerpt of up to 300 characters from the link.
    ```

2. When linking to GitHub, I like to use permanent links to specific commits so that the content won't change. For this one I would probably use <https://github.com/sjsyrek/haskell-study-startup/blob/a9c76592809ddc38d307495f7cd58ceffa587ee2/README.md#readme>.

Reply:
RE: number 2, I think it's the possible the content _will_ change, but not dramatically. Do you prefer links to commits to avoid _code_ that might change, specifically? Would this case be an exception?

Reply:
Whenever possible I prefer linking to how something looks at the time of publication. For example, issue 70 linked to Nanocoin. Rather than linking directly to the project's home page (<https://github.com/tdietert/nanocoin>) I linked to a snapshot of the README (<https://github.com/tdietert/nanocoin/blob/25fe5e8/README.md>). This is probably unnecessary, but I like being able to see how things were when I linked to them.

Reply:
You are a very particular fellow, aren't you? Very well, then.